src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -51,20 +51,22 @@ private NewGameChooser(final Frame owner) {
     setupListeners();
     setWidgetActivation();
     updateInfoPanel();
+    WaitDialog loadingScreen = new WaitDialog(owner, "Loading Maps...");
+    refreshGameList();
+//    SwingUtilities.invokeLater(() -> {loadingScreen.setVisible(false);loadingScreen.dispose();}); dispose() is throwing a "Exception: null" exception
```

Comment:
For whatever reason loadingScreen.dispose() is throwing an error from the substance library...
Any Ideas on how to fix that?
And ignore the SwingUtilities thing this was just for testing... and a temporary workaround